torch/csrc/autograd/saved_variable.h

```diff
@@ -42,7 +42,7 @@ class TORCH_API SavedVariable {
   }
 
   void reset_grad_function() {
-    grad_fn_.reset();
+    // No op
```

Comment:
Is this function ever called?

Reply:
Its used here: https://github.com/pytorch/pytorch/blob/master/tools/autograd/gen_autograd_functions.py#L327

I guess I can remove it in this PR as well?